src/digest/sha1.cr

```diff
@@ -10,9 +10,12 @@ class Digest::SHA1 < Digest::Base
   # This is a direct translation of https://tools.ietf.org/html/rfc3174#section-7
   # but we use loop unrolling for faster execution (about 1.07x slower than OpenSSL::SHA1).
 
+  {% begin %}
+  alias NativeUInt = {% if flag? :bits64 %}UInt32{% else %}UInt64{% end %}
```

Comment:
Should be marked as nodoc

Reply:
Also, is this change from UInt32 to UInt64 a correct thing? I copied the algorithm from somewhere else and it was always using uint32. I wouldn't change that in this PR (I would leave it for a different PR).

Reply:
Or just `private alias`.